### ISSUE TITLE
Screen reader announces checkbox for a menu item in Contextual Menu

### ISSUE THREAD

**Issue by RashmiRanjanParida** [microsoft/fluentui#16142]
### Environment Information

- **Package version(s)**: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
- **Browser and OS versions**:  Edge 

### Describe the issue:
Screen reader announces "checkbox" for the items present in contextual menu, while making canCheck true.

Actual Results:
Incorrect role is defined for menu item present in contextualmenu. Screen reader announces ' Term menu item checkbox 2 of 5'

Expected Result:
Correct role should be define for menu item present in contextualmenu. Screen reader should announce 'Term menu item 2 of 5'

### Please provide a reproduction of the issue in a codepen:
if you go to https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu and select the contextual menu under  "ContextualMenu with checkable menu items and toggleable split button", run the screen reader you can get "Mobile menu item checkbox unchecked 3 of 10"
![image](https://user-images.githubusercontent.com/42003135/101164986-e2cde400-365b-11eb-9c8d-0c9130f67ff6.png)

#### Actual behavior:

screen reader reads "Mobile menu item checkbox unchecked 3 of 10"

#### Expected behavior:

screen reader should read "Mobile menu item 3 of 10"

**Comment by kolaps33** [comment]
I was looking to that and figure out following:
- the keyword "checkbox" narrates only Narrator and not NVDA 
- even "checkbox" looks like correct, because user can check/uncheck menu item

Please could you specify what is wrong about narrating "checkbox"? if you think that "checkbox" keyword should not be narrated at all, please create bug for Narrrator screen reader.
